_j/mkl_threads.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized deployments
+---
+
+Consider adjusting ``OMP_NUM_THREADS`` environment variable for containerized
+deployments, one or more libraries use Intel's MKL that does not detect
+correctly one or more libraries use Intel's MKL that does not detect correctly
```

Comment:
`one or more libraries use Intel's MKL that does not detect correctly` it is repeated